Week1/js-exercise/Round-a-number.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+let z = 7.25;
+console.log(z)
+
+let a = Math.round(z);
+
+console.log(a, z);
+
+let hightest = null;
+
+if (z > a) {
+    hightest = z;
+    console.log(`The hightest number is ${hightest}`);
```

Comment:
you can move this console log outside the if statement to log the highest value, no matter if it was `a` or `z`.